pkg/gui/controllers/branches_controller.go

```diff
@@ -539,12 +539,11 @@ func (self *BranchesController) fastForward(branch *models.Branch) error {
 	message := utils.ResolvePlaceholderString(
 		self.c.Tr.Fetching,
```

Comment:
Forgot to comment on this in a previous iteration: If we're going to shorten it, I'd shorten it to 'Fast-forwarding' given that that's the important thing that's happening.

Reply:
Makes sense, see fc868cdda55b. I also renamed the variable to `FastForwarding`, and adapted the other languages as good as I could (template substitution would have been broken for these, as I changed the placeholders).

Reply:
Looks good. This PR is good to merge!

As an aside, it would be cool to have some chatGPT powered translation tool. I hear chatGPT does a very good job at translating, far better than we could do with google translate. There's definitely a startup idea there